keras_cv/datasets/waymo/transformer.py

```diff
@@ -0,0 +1,635 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Transformer to convert Waymo Open Dataset proto to model inputs."""
+
+from typing import Dict
+from typing import List
+from typing import Sequence
+from typing import Tuple
+
+import numpy as np
+import tensorflow as tf
+from waymo_open_dataset import dataset_pb2
```

Comment:
If we're going to make WOD an optional dependency, we need to wrap our imports in a try/catch to allow importing the package without throwing an error due to missing deps.

(Here and in load.py)

Reply:
Added try/catch to __init__.py.